utils/puck/textPage.ts

```diff
@@ -0,0 +1,136 @@
+export const textPage = {
```

Comment:
Why do we need this file? 
Something specific to puck config?

Reply:
This is one of the example pages. I didn't want to have one huge file with many example pages, so I created file per page. Bad thing is that we have to copy the output of the editor to the codebase (until we have DB).